src/rust_slice.rs

```diff
@@ -0,0 +1,84 @@
+// From https://raw.githubusercontent.com/rust-lang/rust/1.56.1/library/core/tests/slice.rs
+// (which has not changed from 1.56.1 to 1.67.1).
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Not sure why this was disabled for wasm in the stdlib tests. We should try it on wasm and delete this if it works.